packages/bundler-plugin-core/src/utils.ts

```diff
@@ -264,6 +264,24 @@ export function generateGlobalInjectorCode({
   return code;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function generateModuleMetadataInjectorCode(metadata: any) {
+  // The code below is mostly ternary operators because it saves bundle size.
+  // The checks are to support as many environments as possible. (Node.js, Browser, webworkers, etc.)
+  return `
+    var _global2 =
+      typeof window !== 'undefined' ?
+        window :
+        typeof global !== 'undefined' ?
+          global :
+          typeof self !== 'undefined' ?
+            self :
+            {};
+
+    _global2.__MODULE_METADATA__ = _global2.__MODULE_METADATA__ || {};
+    _global2.__MODULE_METADATA__[new Error().stack] = ${JSON.stringify(metadata)};`;
```

Comment:
https://github.com/getsentry/sentry-javascript/pull/8438#discussion_r1250736786

In case we change this property name we should align these two PRs.